lib/cc/cli/engines/validate.rb

```diff
@@ -0,0 +1,36 @@
+require "cc/analyzer"
+
+module CC
+  module CLI
+    module Engines
+      class Validate < EngineCommand
+        def run
+          if invalid_engines.any?
+            warn
+            true
```

Comment:
I don't think we have that exit code behavior currently, so if so we should do that separately. Probably want a CommandResult object so we don't litter exit calls in the classes